src/main/java/com/redhat/cloud/custompolicies/app/model/FullTrigger.java

```diff
@@ -51,6 +56,26 @@ public FullTrigger(Policy policy) {
     Condition cond = new Condition();
     cond.expression = policy.conditions;
     conditions.add(cond);
+    storeActions(trigger, policy);
+  }
+
+  private void storeActions(Trigger trigger, Policy policy) {
+    String[] actionsIn = policy.actions.split(";");
```

Comment:
This will fail if `actions` is `null`. If is empty string, it will throw later as this will yield an array of one element `""`

Reply:
Good catch!